crates/bevy_transform/src/components/global_transform.rs

```diff
@@ -16,10 +17,10 @@ use bevy_reflect::{std_traits::ReflectDefault, Reflect};
 ///
 /// ## [`Transform`] and [`GlobalTransform`]
 ///
-/// [`Transform`] is the position of an entity relative to its parent position, or the reference
-/// frame if it doesn't have a [`Parent`](bevy_hierarchy::Parent).
+/// [`Transform`] is the local position of an entity in space relative to its parent position,
```

Comment:
The link to `Transform` is broken because there is a link defined to `transform` for the "Example" section.

The link to `global_vs_local_translation` in the same section is also broken (I'm not sure if the example was simply removed or renamed)